bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gmos/GmosRule.java

```diff
@@ -1850,6 +1855,33 @@ public IConfigMatcher getMatcher() {
     };
 
     /**
+     * REL-2456: We request a Phase II check to give a warning if the DTA-X
+     * position is not a multiple of the y-pixel binning.
+     */
+    private static IConfigRule DTA_X_Y_MULTIPLE_BINNING_RULE = new IConfigRule() {
+        private static final String errMsg = "The DTA X offset (%d) should be multiple of the Y-binning (%d)";
+
```

Comment:
I don't think we need any new public methods.  We can define this id and two others, in the other rule being tested by this same test case:

````java
   public static IConfigRule GMOS_S_DTA_X_RULE = new IConfigRule() {
       public static final IdY1   = PREFIX + "Y1";
       public static final IdY2_4 = PREFIX + "Y2_4";
       ...
   }
````